elements/lisk-codec/src/types.ts

```diff
@@ -71,3 +71,8 @@ export interface CompiledSchemas {
 export interface Validator {
 	addMetaSchema: (schema: object, key?: string) => {};
 }
+
+export interface IteratableGenericObject extends GenericObject {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	[Symbol.iterator](): Iterator<{ key: string; value: any }>;
```

Comment:
```suggestion
	[Symbol.iterator](): Iterator<IteratorReturnValue>;
```

Reply:
Had to do it locally as I had to move the IteratorReturnValue to the types file.